src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1649,6 +1649,13 @@ namespace ScheduleManager {
                 FileIntervalInterpolated = true;
             }
 
+            if (lAlphaBlanks(6)) Alphas(6) = "YES";
+            if ((Alphas(6)) == "YES") {
+                state.dataScheduleMgr->Schedule(SchNum).UseDaylightSaving = true;
```

Comment:
Nit picking. But the first 2 lines here are not necessary. Start with line 1654, then reset it if Alphas(6) = No.

Reply:
Addressed per comment.